plugins/register-component/README.md

```diff
@@ -1,5 +1,7 @@
 # Register component plugin
 
+> This plugin is deprecated in favor of [`@backstage/catalog-import`](https://github.com/backstage/backstage/tree/master/plugins/catalog-import), and will be soon removed from the project.
```

Comment:
Let's add a changeset so we get this update released and displayed on NPM as well